### PR TITLE
Telemetrystart

### PR DESCRIPTION
**Pull request by kb1lqc**
I added a telemetry quickstart page just prior to having the user learn to command an LED on. Additionally I updated some formatting of the guide to account for generic `REPLACEME` text in the example files.

Code formatting wasn't accounted for in this update. This was also one of the last items we learned about to fix when sitting down with Jake going through the learning process.

> Note that this pull request directly stems from my Pull Request #84 so I believe that those files should go away as not needing review once #84 is merged.

### CODE REVIEW

File: Tutorials/start/configuring-faraday.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The application files are located:
 
 ## Device Configuration Setup
 
-`Deviceconfiguration.py` uses two .ini files, one to talk to Proxy and one to program Faraday hardware with. First we should ensure the program can communicate with Proxy running in the background. For this we need to edit the `[DEVICES]` section of `deviceconfiguration.ini` to contain the same configuration Proxy is configured with in `proxy.ini`
+First we should ensure the Device Configuration program can communicate with Proxy running in the background. For this we need to edit the `[DEVICES]` section of `deviceconfiguration.ini` to contain the same configuration Proxy is configured with in `proxy.ini`
 
 `[FLASK]`: Flask server configuration values
```
Comment (kb1lqd) [inline Tutorials/start/configuring-faraday.md:17]: Can more than one unit be used at a time? If not then this is confusing (multiple unit count in INI) unless explicitly stated.

**Review by kb1lqd — COMMENTED** [review]
@kb1lqc should you include a non-5 minute example? After reading this I don't know howto get the last... 15 minutes.?

File: Tutorials/start/configuring-faraday.md (hunk 1 above)
Comment (kb1lqc) [reply Tutorials/start/configuring-faraday.md:17]: Not sure what you're talking about @kb1lqd? This text doesn't comment on multiple units and `deviceconfiguration.ini` only contains configuration for a single unit.

**Review by kb1lqd — DISMISSED** [review]
@kb1lqc  Can you add in the START table of contents links for telemetry and APRS to give a better forwarning? Maybe a sub-section in the TOC like "Applications"

image

File: Tutorials/start/configuring-faraday.md (hunk 1 above)
Comment (kb1lqd) [reply Tutorials/start/configuring-faraday.md:17]: Disregard, discussed in #84

**Comment by kb1lqc** [comment]
@kb1lqd I added new links to the Telemetry getting started in the TOC on the start page. Additionally, I wrote a getting started with applications section directly after the Table of Contents including an image of APRS operation using Faraday. Please review and approve if you are OK with this.

**Review by kb1lqd — APPROVED** [review]
Requested updates performed. Good to merge.

**Pull request merged**